app/src/main/java/org/jellyfin/androidtv/ui/itemdetail/FullDetailsActivity.java

```diff
@@ -1260,6 +1276,18 @@ public void onClick(View v) {
                 more.show();
             }
         });
+        if (userPreferences.getValue().get(UserPreferences.Companion.getVideoPlayer()) == PreferredVideoPlayer.CHOOSE) {
```

Comment:
Probably easier to add an `else` statement when checking if the play button should be shown